_data/contacts.yml

```diff
@@ -20,20 +20,21 @@
       title: Writer/Editor, Director's Office
 - department: Agency for Healthcare Research and Quality
   website: https://www.ahrq.gov/policy/electronic/plain-writing/index.html
-  contacts: 
+  contacts:
     - name: Cindy Brach
       email: Cindy.Brach@ahrq.hhs.gov
 - department: Consumer Finance Protection Bureau
   website: http://www.consumerfinance.gov/plain-writing/
   officials:
-    - name: Gena Chieco
-      title: Executive Secretary and Counsel to the Director
-  contacts: CFPB_Plain_Writing_Act@cfpb.gov
+    - name: Jocelyn Sutton
+      title: Deputy Chief of Staff
+  contacts: CFPB_Plain_Writing_Act@consumerfinance.gov
```

Comment:
Contacts is listed here, but doesn't appear in [preview](https://federalist-dad55c1b-07c6-4c7e-89da-5aff1f9335fd.app.cloud.gov/preview/gsa/plainlanguage.gov/lgrabuck-patch-2/law/agency-programs/)

![image](https://user-images.githubusercontent.com/3385219/185180036-b85c514b-0307-4ae9-93f8-66e5b061c5a0.png)

This was happening before. @ToniBonittoGSA do you know if this is intentional or should we create an issue to list contact if **only** email is provided?

Reply:
It's broken b/c they didn't follow the right format; will fix. Thx for finding that!